### PR TITLE
Update observability info

### PR DESCRIPTION
**Pull request by dockerymick**
https://github.com/stolostron/backlog/issues/19936

### CODE REVIEW

File: access_control/rbac.adoc [hunk 1]
```diff
@@ -539,12 +539,35 @@ View the following console and API RBAC tables for governance lifecycle:
 [#observability-RBAC]
 === Observability RBAC
 
-To use the observability features, you must be assigned the `cluster-admin` or the `open-cluster-management:cluster-manager-admin` role. View the following list of observability features:
+To use the observability features, you must be assigned the `view` role. View the following list of observability features:
```
Comment (ckandag) [inline access_control/rbac.adoc]: I think this should be

To view  the observability metrics for a managed cluster, you must have  `view` access to that managed cluster on the Hub.

File: access_control/rbac.adoc [hunk 2]
```diff
@@ -539,12 +539,35 @@ View the following console and API RBAC tables for governance lifecycle:
...
 
-//To access the managed cluster metrics run the following command:
 * Access managed cluster metrics.
++
+Users are denied access to managed cluster metrics, if they are not assigned to the `view` role in the same namespace of the managed cluster.
```
Comment (ckandag) [inline access_control/rbac.adoc]: rewrite as below


Users are denied access to managed cluster metrics, if they are not assigned to the `view` role to the managed cluster's namespace on the Hub.

**Review by ckandag — CHANGES_REQUESTED** [review]
comments inline

**Comment by dockerymick** [comment]
@ckandag thanks for your review and comments.

File: access_control/rbac.adoc [hunk 3]
```diff
@@ -539,11 +539,11 @@ View the following console and API RBAC tables for governance lifecycle:
 [#observability-RBAC]
 === Observability RBAC
 
-To use the observability features, you must be assigned the `view` role. View the following list of observability features:
+To view the observability metrics for a managed cluster, you must have `view` access to that managed cluster on the hubb cluster. View the following list of observability features:
```
Comment (ckandag) [inline access_control/rbac.adoc]: hubb ->. hub

**Comment by dockerymick** [comment]
@ckandag Thanks again for the review! Merging this

**Pull request merged**